src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+// Action Types
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- According to the project requirements, the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

Reply:
@mckent05 Done. Thanks a lot!